portfolio/src/main/webapp/index.html

```diff
@@ -17,11 +17,16 @@ <h2><center>About Me</center></h2>
             💛💙AGGIE PRIDE!!💙💛</center></p>
         <p><center>Click <a href="AboutMe.html">here</a> to learn a little more.<center></p>
         <p><center>Click <a href="MyGallery.html">here</a> to see some pictures!<center></p>
-        
-        <h1>Hello Nzinga button</h1>
-        <p>Click here:</p>
-        <button onclick="getHelloNzingaUsingAsyncAwait()">Fetch Hello Nzinga!</button>
+
+        <form action="/data" method="POST">
+            <p>Leave a comment!</p>
+            <input type="textarea" name="textInput">
+            <br/><br/>
+            <input type="submit" />
+        </form>
+
         <div id="HelloNzinga-container"></div>
+        <script> getHelloNzingaUsingAsyncAwait() </script>
```

Comment:
An alternative to this would be to set the `body.onload` HTML attribute to `getHelloNzingaUsingAsyncAwait()`